src/Propel/Generator/Command/DataDictionaryExportCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * MIT License. This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Propel\Generator\Command;
+
+use Propel\Generator\Manager\DataDictionaryExportManager;
+use Symfony\Component\Console\Exception\MissingInputException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Charles Crossan <crossan007@gmail.com>
+ */
+class DataDictionaryExportCommand extends AbstractCommand
+{
+    public const DEFAULT_OUTPUT_DIRECTORY = 'generated-datadictionary';
+
+    /**
+     * {@inheritDoc}
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->addOption('output-dir', null, InputOption::VALUE_REQUIRED, 'The output directory', self::DEFAULT_OUTPUT_DIRECTORY)
```

Comment:
Would be better to move `output-dir` to const, since this value is used in several places.

Reply:
You can also use `static::` here for class const access

Reply:
Ok, extracted the option name into a const.

I don't think it matters here if I use `static::` or `self::`, as I can't think of a scenario where the value is overridden by a child class (which is the only difference between self and static, right?). I changed it anyway.